apps/ui/src/hooks/Communities/useUserCommunitiesStatus.tsx

```diff
@@ -1,26 +1,18 @@
-import { UserStatus } from '@communecar/types';
-
-import { getRandomOption } from '../../utils';
-import { useGetAllCommunities } from './useGetAllCommunities';
 import { UserCommunitiesStatus } from '../../types/community-type';
+import { fetchUserCommunitiesStatus } from '../../apis/communities/fetch-community-user-status';
+import { useQuery } from '@tanstack/react-query';
 
-const userStatusOptions: UserStatus[] = Object.values(
-  UserStatus,
-) as UserStatus[];
-
-const useUserCommunitiesStatus = (userId: number): UserCommunitiesStatus => {
-  const { data: communities } = useGetAllCommunities();
-  console.log({ userId }); //TODO when the server ready
-
-  const communitiesStatus: UserCommunitiesStatus = {};
+const useUserCommunitiesStatus = (
```

Comment:
why won't you return isLoading or error to the user of the hook? I think you can just return this useQuery as it is so it can be used in multiple places